docs/source/guides/installation_guide.rst

```diff
@@ -0,0 +1,309 @@
+
+
+Here's a one-liner to install the above packages on linux:
+``sudo apt-get install python3-dev build-essential libffi-dev python3-pip``
+
+Standard Installation
+---------------------
+
+``nucypher`` can be installed by ``pip`` or ``pipenv``\ , or run with ``docker``.\
+Ensure you have one of those installation tools installed for you system:
+
+
+* `Pip Documentation <https://pip.pypa.io/en/stable/installing/>`_
+* `Pipenv Documentation <https://pipenv.readthedocs.io/en/latest/>`_
+* `Docker <https://docs.docker.com/install/>`_
+
+Pip Installation
+^^^^^^^^^^^^^^^^
+
+In order to isolate global system dependencies from nucypher-specific dependencies, we *highly* recommend
+using ``python-virtualenv`` to install ``nucypher`` inside a dedicated virtual environment.
+
+For full documentation on virtualenv see: https://virtualenv.pypa.io/en/latest/
+
+Here is the recommended procedure for setting up ``nucypher`` in this fashion:
+
+. Create a Virtual Environment
+
+   .. code-block:: bash
+
+       $ virtualenv /your/path/nucypher-venv
+       ...
+
+    Activate the newly created virtual environment:
+
+   .. code-block::
+
+       $ source /your/path/nucypher-venv/bin/activate
+       ...
+       $(nucypher-venv)
+
+    ``note:: Successful virtualenv activation is indicated by '(nucypher-venv)$' prepended to your console's prompt``
+
+
+. Install Application Code with Pip
+
+   .. code-block:: bash
+
+       $(nucypher-venv) pip3 install -U nucypher
+
+. Verify Installation
+
+    Before continuing, verify that your ``nucypher`` installation and entry points are functional;
+    Activate your virtual environment (if you haven't already) and run the ``nucypher --help`` command in the console:
+
+    .. code-block:: bash
+
+       nucypher --help
+
+    You will see a list of possible usage options (\ ``--version``\ , ``-v``\ , ``--dev``\ , etc.) and commands (\ ``status``\ , ``ursula``\ , etc.).
+    For example, you can use ``nucypher ursula destroy`` to delete all files associated with the node.
+
+    .. code-block::
+
+       In Python:
+
+       ```python
+       import nucypher
+       ```
+
+Pipenv Installation
+^^^^^^^^^^^^^^^^^^^
+
+. Install Application code with Pipenv
+
+   Ensure you have ``pipenv`` installed (See full documentation for pipenv here: `Pipenv Documentation <https://pipenv.readthedocs.io/en/latest/>`_\ ).
+   Then to install ``nucypher`` with ``pipenv``\ , run:
+
+.. code-block:: bash
+
+   $ pipenv install nucypher
+
+. Verify Installation
+
+In the console:
+
+.. code-block:: bash
+
+   $ nucypher --help
+
+In Python:
+
+.. code-block:: python
+
+  import nucypher
+
+
+Docker Installation
+-------------------
+
+. Install `Docker <https://docs.docker.com/install/>`_
+. (Optional) Follow these post install instructions: `https://docs.docker.com/install/linux/linux-postinstall/ <https://docs.docker.com/install/linux/linux-postinstall/>`_
+. Get the latest nucypher image:
+
+   ``docker pull nucypher/nucypher:latest``
```

Comment:
Did you intend for this to instead be a `.. code-block::`?

Reply:
:heavy_check_mark: